libc-test/build.rs

```diff
@@ -333,6 +356,12 @@ fn main() {
             "ptrace" |
             "sigaltstack" if rumprun => true,
 
+            // uses size_t instead of socklen_t
+            "getnameinfo" if dragonfly => true,
+
+            // is defined elsewhere
+            "__dfly_error" if dragonfly => true,
```

Comment:
exactly :). I am working on a patch to support thread-local externs, so that we don't need that anymore.